### DISCUSSION TITLE
Change conversation state in another chat

### DISCUSSION THREAD

**Discussion by yuretzgt** [python-telegram-bot/python-telegram-bot#3348] [Q&A]
I use my bot for making communication between users where bot between then, e.g. customer support

So there is few states

action_user
conversation_user
conversation_support
so, them my support user sending a message to bot, i need send another message to user (i know chat_id with him) and change user's current state

Of course i can add a button to message, and press on it will change the state, but i hope there is more beautiful solution

So, the question is, how to change conversation state by chat.id?

**Comment by Bibo-Joshi** [comment]
This question was also asked at https://stackoverflow.com/questions/74345746. Please don't ask a questions at mutliple places at once.

**Comment by yuretzgt** [comment]
Any suggestions? )
